src/server/controllers/account-transactions.ts

```diff
@@ -717,6 +753,7 @@ const accountTransactions: RequestHandler<Params> = async (req, res) => {
         const mapping = pick(csvMapping, fields);
 
         const mappedTransactions = result.transactions.nodes.map((t) => applyMapping(mapping, t));
+        console.log(result.transactions.nodes[0], mappedTransactions[0]);
```

Comment:
Is the `console.log` needed for debug or leftover?

Reply:
Definitely leftover, thanks for the catch